vars/ciK8sPushHelm.groovy

```diff
@@ -16,10 +17,10 @@ def call(project, chartVersion, museumAddr, replaceTag = false, failIfExists = f
         }
 
         if (replaceTag) {
-            ciUpdateYamlFile(
```

Comment:
`ciUpdateYamlFile`  is a nice method, but seems like its not working for nested yaml properties like `image.tag` , so i will need to try to make it work later to reuse everywhere when we are updating yaml files. But for now, we will do a bit of code repetition